File: 2011-final-value-of-variable-after-performing-operations/2011-final-value-of-variable-after-performing-operations.java
class Solution {
    public int finalValueAfterOperations(String[] operations) {
        int x = 0;

        for (int i = 0; i < operations.length; i++) {
            switch (operations[i]){
                case "++X", "X++":
                    x += 1;
                    break;
                case "--X", "X--":
                    x -= 1;
                    break;
            }
        }
        return x;
    }
}

File: 2011-final-value-of-variable-after-performing-operations/2011-final-value-of-variable-after-performing-operations.js
/**
 * @param {string[]} operations
 * @return {number}
 */
var finalValueAfterOperations = function(op) {
    let ans = 0;
    
    op.forEach((str)=> {
        if(str === "++X" || str === "X++") ans++;
        if(str === "--X" || str === "X--") ans--;
    })
    return ans;
};